paraatm/simulation_method/vcas.py

```diff
@@ -0,0 +1,380 @@
+"""
+NASA NextGen NAS ULI Information Fusion
+
+@organization: Arizona State University
+@author: Yuhao Wang
+@date: 2020-04-19
+
+This Python script is used for VCAS (Voice Communication-Assisted Simulation)
+Currently monitors altitude related commands during descend only
+The script outputs aircraft trajectory based on input flight plan and text command translated from audio
+Simulation based on NATS beta1.7 standalone version
+The script can also check compliance for each command
+"""
+
+import time
+import os
+import io
+from shutil import copyfile
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import scipy.stats as sts
```

Comment:
This is the first use of `scipy` in para-atm.  Please add scipy as a dependency to `setup.py`.  This will ensure that scipy gets installed automatically when people install para-atm.